cluster/points_writer.go

```diff
@@ -196,6 +202,13 @@ func (w *PointsWriter) WritePoints(database, retentionPolicy string, consistency
 	w.statMap.Add(statWriteReq, 1)
 	w.statMap.Add(statPointWriteReq, int64(len(points)))
 
+	if w.MaxConcurrentWrites > 0 {
```

Comment:
This isn't quite what the intent of #6559 was intended to solve.  #6559 is to return an error on the HTTP connection _before_ getting into `PointsWriter` or parsing points.  It needs to drop the connection before doing all the work to get down to this level.